crates/octez/src/async/client.rs

```diff
@@ -516,4 +592,58 @@ mod test {
         let res = OctezClient::extract_digits(input);
         assert!(res.is_ok_and(|balance| balance == 15.23453));
     }
+
+    #[test]
+    fn parse_contract_address() {
+        let contract_hash = "KT1F3MuqvT9Yz57TgCS3EkDcKNZe9HpiavUJ";
+        assert_eq!(
+            super::parse_contract_address(&format!(
```

Comment:
Isn't `parse_contract_address` already imported in `use::super::*`? There shouldn't be a need to 
use the full qualified name. 

Applicable to the other tests below

Reply:
The test case itself is named `parse_contract_address` and thus collides with the imported method. I've renamed those test cases even though I don't think prefixing `super::` matters that much here.